public/app/features/dashboard/components/GenAI/utils.test.ts

```diff
@@ -98,22 +98,22 @@ describe('getDashboardChanges', () => {
 });
 
 describe('isLLMPluginEnabled', () => {
-  it('should return true if LLM plugin is enabled', async () => {
-    // Mock llms.openai.health to return true
-    jest.mocked(llms.openai.health).mockResolvedValue({ ok: true, configured: false });
+  it('should return false if LLM plugin is not enabled', async () => {
+    // Mock llms.openai.health to return false
+    jest.mocked(llms.openai.health).mockResolvedValue({ ok: false, configured: false });
 
     const enabled = await isLLMPluginEnabled();
 
-    expect(enabled).toBe(true);
+    expect(enabled).toBe(false);
   });
 
-  it('should return false if LLM plugin is not enabled', async () => {
-    // Mock llms.openai.health to return false
-    jest.mocked(llms.openai.health).mockResolvedValue({ ok: false, configured: false });
+  it('should return true if LLM plugin is enabled', async () => {
+    // Mock llms.openai.health to return true
+    jest.mocked(llms.openai.health).mockResolvedValue({ ok: true, configured: false });
 
     const enabled = await isLLMPluginEnabled();
 
-    expect(enabled).toBe(false);
+    expect(enabled).toBe(true);
   });
```

Comment:
These changes are just switching the order of tests to first check false case then true given newly added cache